tensorflow_quantum/core/ops/tfq_ps_weights_from_symbols_op.cc

```diff
@@ -146,7 +146,7 @@ class TfqPsWeightsFromSymbolOp : public tensorflow::OpKernel {
     auto DoWork2 = [&](int start, int end) {
       for (int i = start; i < end; i++) {
         for (int j = 0; j < n_symbols; j++) {
-          for (int k = 0; k < output_results.at(i).at(j).size(); k++) {
+          for (size_t k = 0; k < output_results.at(i).at(j).size(); k++) {
```

Comment:
Why do just k and not i, j as well ?

Reply:
Ah, that's because k is the only one being compared to a `.size()` value; i and j are used as indexes.